user_docs/en/userGuide.md

```diff
@@ -2516,6 +2516,11 @@ This option is only available if reporting of spelling errors is enabled in NVDA
 This option allows the user to control if key presses generated by applications such as on-screen keyboards and speech recognition software should be processed by NVDA.
 This option is on by default, though certain users may wish to turn this off, such as those typing Vietnamese with the UniKey typing software as it will  cause incorrect character input.
 
+##### Maximum delay between two key presses for a repeated key press {#MaxRepeatedKeyDelay}
```

Comment:
The Rabbit suggests to improve this paragraph:
* Mention the default value and the range of acceptable values.
* Provide an example scenario where increasing the delay might be beneficial.
* Ensure consistency with the rest of the document by using a similar structure for settings 

The second point does not seem interesting to me: it just makes the documentation more verbose.
Though, the first and third point seem interesting since we are now documenting quite systematically the possible values and the default for each new option. For now we have only documented checkboxes / binary flags (Enabled/disabled) and combo-boxes / list of multiple choices.
Do we want to generalize this? If yes, how? Or is there a risk that the user becomes always more a technical documentation. E.g.:
Value range, value type (boolean, integer, list of strings), default value...

Cc @seanbudd, @LeonarddeR, @Adriani90 for opinion

Reply:
I would not be too verbose here. I only agree with the rabbit's point number one. Mentioning the default value, and how it translates to seconds for the math-averse, seems like all we should do. I would not get overly formal, either.